Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM rust:latest as builder
+
+WORKDIR /app
+
+COPY . .
+
+# Build the server
+RUN cargo build --release
+
+# Use a minimal image for running the server
+FROM ubuntu
+
+RUN apt update && apt install -y libpq5
```

Comment:
*Just a suggestion*
I think you could use this `apt update && apt install -y libpq5 && rm -rf /var/lib/apt/lists/` instead.

Why: After installing packages, it's a good practice to remove the apt cache to reduce the image size. It actually ensures that unnecessary metadata and package lists are cleaned up.